catalog/ui/catalog-ui-search/webpack/config/devServer.js

```diff
@@ -0,0 +1,54 @@
+var path = require('path');
+var webpack = require('webpack');
+var merge = require('webpack-merge');
+
+var dev = require('./dev');
```

Comment:
Is there a reason why we can't keep the webpack configs combined?

Reply:
Yes, I had to split them up to prevent hot reloading from getting into quick builds, since that would be misleading.